components/EditISCNMetadataModal.vue

```diff
@@ -110,15 +110,17 @@ const iscnData = ref({
   coverUrl: ''
 })
 
-const { payload } = useISCN(iscnData)
+const iscnOnchainData = ref({} as any)
```

Comment:
```suggestion
const iscnChainData = ref({} as any)
```
or `iscnOldData` `iscnExistingData`, or even just `iscnData` in this scope of component